capstone/src/main/java/com/google/sps/data/SentimentData.java

```diff
@@ -0,0 +1,15 @@
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+
+public class SentimentData{
+    public double getSentiment(String message){
+    Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
```

Comment:
Creating a new client every time the getSentiment method is called is pretty expensive. The way this class is set up now, it accepts one string and outputs a sentiment score. Since we have over 200 responses, you would have to create and close the client 200+ times . One recommendation is to initialize the client in the constructor and close the client in the destructor so that only one client is initialized and used for 200+ messages